src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1127,6 +1127,8 @@ namespace NServiceBus.Config
         [System.Configuration.ConfigurationPropertyAttribute("Threshold", DefaultValue="Info", IsRequired=true)]
         public string Threshold { get; set; }
     }
+    [System.ObsoleteAttribute("Please use `Switch to the code API by using \'EndpointConfiguration.EnlistWithLega" +
```

Comment:
Actually you usually use `Please` as it is approved.txt. But it looks wrong as I used:
`ReplacementTypeOrMember` instead of `Message`.